### ISSUE TITLE
TF 2.0 TPUStrategy error 'explicit_paddings' not in Op<name=Conv2D; ...>

### ISSUE THREAD

**Issue by thatsmesasha** [tensorflow/tensorflow#27126]
**System information**
- TensorFlow version: 2.0.0-alpha0
- Python version: 3.6
- Environment: Google Collaboratory with TPU

**Describe the current behavior**
I'm trying to do TPUStrategy for tensorflow 2.0 with disabled eager execution. I'm using Keras for the CNN model. When I try to run the .fit function on the model, I get an error:

NodeDef mentions attr 'explicit_paddings' not in Op<name=Conv2D; signature=input:T, filter:T -> output:T; attr=T:type,allowed=[DT_HALF, DT_BFLOAT16, DT_FLOAT, DT_DOUBLE]; attr=strides:list(int); attr=use_cudnn_on_gpu:bool,default=true; attr=padding:string,allowed=["SAME", "VALID"]; attr=data_format:string,default="NHWC",allowed=["NHWC", "NCHW"]; attr=dilations:list(int),default=[1, 1, 1, 1]>; NodeDef: {{node conv2d/Conv2D}}. (Check whether your GraphDef-interpreting binary is up to date with your GraphDef-generating binary.).

**Code to reproduce the issue**
Google Collaboratory notebook: https://gist.github.com/thatsmesasha/0b7cca49517987166f479382158d2d42

**Comment by dynamicwebpaige** [comment]
TPUs are not yet supported with TensorFlow 2.0.
You can track [this issue](https://github.com/tensorflow/tensorflow/issues/24412) to stay up-to-date on progress.

**Comment by ymodak** [comment]
Closing this issue since we already have a tracking issue. Feel free to reopen if have any questions. Thanks!

**Comment by ixtiyoruz** [comment]
I am having this problem with TensorFlow 1.13.1 while trt.create_inference_graph
